src/renderers/shared/stack/reconciler/ReactChildReconciler.js

```diff
@@ -27,8 +28,9 @@ function instantiateChild(childInstances, child, name) {
       keyUnique,
       'flattenChildren(...): Encountered two children with the same key, ' +
       '`%s`. Child keys must be unique; when two children share a key, only ' +
-      'the first child will be used.',
-      KeyEscapeUtils.unescape(name)
+      'the first child will be used.%s',
+      KeyEscapeUtils.unescape(name),
+      ReactComponentTreeDevtool.getCurrentStackAddendum(childInstances[name])
```

Comment:
Yeah it makes sense to me. I thought the issue with `Unknown` was about the name of the node but I actually got confused by element vs. instance here. Thanks!